src/components/nve-combobox/nve-combobox.component.ts

```diff
@@ -0,0 +1,365 @@
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import styles from './nve-combobox.styles';
+import { html, LitElement } from 'lit';
+import { ref, createRef } from 'lit/directives/ref.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { customElement, property, state } from 'lit/decorators.js';
+import '../nve-popup/nve-popup.component';
+import '../nve-input/nve-input.component';
+import '../nve-tag/nve-tag.component';
+import '../nve-menu/nve-menu.component';
+import '../nve-menu-item/nve-menu-item.component';
+import '../nve-option/nve-option.component';
+import '../nve-icon/nve-icon.component';
+
+interface Option {
+  label: string;
+  value: string | number;
+  selected?: boolean;
+}
+
+@customElement('nve-combobox')
+export default class NveCombobox extends LitElement implements INveComponent {
+  @property() label?: string;
+  @property() placeholder?: string;
+  @property() requiredLabel?: string = '*Obligatorisk';
+  @property() errorMessage?: string = '-!ERROR!-';
+
+  @property({ type: Array<Option> }) values: Option[] = []; // Liste med verdier som kan velges
+
+  @property({ reflect: true, type: Boolean }) singular: boolean = false;
+  @property({ reflect: true, type: Boolean }) disabled: boolean = false;
+  @property({ reflect: true, type: Boolean }) filled: boolean = false;
+  @property({ reflect: true, type: Boolean }) required: boolean = false;
+  @property({ reflect: true, type: String }) testId: string | undefined = undefined;
+
+  @state() listWithSearchHits: Option[] = []; // Listen som vises inne i popupen når man har skrevet i input feltet
+  @state() displaySearchResult: boolean = false; // Skal listWithSearchHits vises eller ikke
+
+  @state() selectedOptions: Option[] = []; // Valgte alternativer
```

Comment:
Se kommentar om `values` på linje 28. Forøvrig synes jeg det gir mening med dette variabelnavnet om verdiene man kunne velge het `options`...:)